main.py

```diff
@@ -61,9 +61,13 @@ def change_read():
 
     book_id = ui.get_book_id()
     book = store.get_book_by_id(book_id)  
-    new_read = ui.get_read_value()     
-    book.read = new_read 
-    book.save()
+
+    if book != None: # Wyatt: If book value is not None, then the changes are saved
```

Comment:
Python style suggests

`if book is not None:`

rather than 

`if book != None:`

but it's correct either way.